build/Versions.props

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <!-- Repo Version Information -->
+  <PropertyGroup>
+    <VersionBase>2.0.3</VersionBase>
+    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
+  </PropertyGroup>
+
+  <!-- Repo Toolset Features -->
+  <PropertyGroup>
+    <UsingToolMicrosoftNetCompilers>false</UsingToolMicrosoftNetCompilers>
+    <UsingToolSourceLink>false</UsingToolSourceLink>
+    <UsingToolVSSDK>true</UsingToolVSSDK>
+  </PropertyGroup>
+
+  <!-- Toolset Dependencies -->
+  <PropertyGroup>
+    <DotNetCliVersion>2.0.2-vspre-006963</DotNetCliVersion>
+    <MicrosoftDotNetBuildTasksFeedVersion>1.0.0-prerelease-01929-02</MicrosoftDotNetBuildTasksFeedVersion>
+    <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha52</RoslynToolsMicrosoftRepoToolsetVersion>
+    <RoslynToolsMicrosoftSignToolVersion>0.3.7-beta</RoslynToolsMicrosoftSignToolVersion>
+    <SourceLinkCreateCommandLineVersion>2.5.0</SourceLinkCreateCommandLineVersion>
+    <VSWhereVersion>2.2.7</VSWhereVersion>
+  </PropertyGroup>
+
+  <!-- Production Dependencies -->
+  <PropertyGroup>
+    <MicrosoftBuildFrameworkVersion>15.2.0-preview-000093-02</MicrosoftBuildFrameworkVersion>
+    <MicrosoftBuildUtilitiesCoreVersion>15.2.0-preview-000093-02</MicrosoftBuildUtilitiesCoreVersion>
+    <MicrosoftExtensionsDependencyModelVersion>2.0.0-preview1-001960</MicrosoftExtensionsDependencyModelVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
+    <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
+    <NuGetBuildTasksPackVersion>4.4.0-preview3-4475</NuGetBuildTasksPackVersion>
+    <NuGetPackagingVersion>4.4.0-preview3-4475</NuGetPackagingVersion>
+    <NuGetProjectModelVersion>4.4.0-preview3-4475</NuGetProjectModelVersion>
+    <PlatformAbstractionsVersion>2.0.0-preview1-001960</PlatformAbstractionsVersion>
+    <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
+    <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
+  </PropertyGroup>
+
+  <!-- Test Dependencies -->
+  <PropertyGroup>
+    <DotNetXUnitVersion>2.3.0</DotNetXUnitVersion>
```

Comment:
>DotNetXUnitVersion [](start = 5, length = 18)

Not necessary. The toolset defines default XUnit version

Reply:
This is newer than what toolset uses by default.

Reply:
The toolset uses 2.3.0:
```
<XUnitVersion Condition="'$(XUnitVersion)' == ''">2.3.0</XUnitVersion>
<XUnitRunnerConsoleVersion Condition="'$(XUnitRunnerConsoleVersion)' == ''">$(XUnitVersion)</XUnitRunnerConsoleVersion>
<XUnitRunnerVisualStudioVersion Condition="'$(XUnitRunnerVisualStudioVersion)' == ''">2.3.0</XUnitRunnerVisualStudioVersion>
```

Reply:
Actually, ```DotNetXUnitVersion``` doesn't seem to be used at all.

Reply:
The Core runner is in the same package as desktop runner.

Reply:
So we don't need tow different versions.

Reply:
Fixed.